### PR TITLE
[TACHYON-794] Added test coverage for inodeFile

### PR DESCRIPTION
**Pull request by yupeng9**
https://tachyon.atlassian.net/browse/TACHYON-794

### CODE REVIEW

File: servers/src/test/java/tachyon/master/next/filesystem/meta/InodeFileTests.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  }
+
+  @Test
+  public void getBlockSizeBytesTest() {
+    InodeFile inode1 = createInodeFile(1);
+    Assert.assertEquals(Constants.KB, inode1.getBlockSizeBytes());
+  }
+
+  @Test
+  public void isFullyInMemoryTest() throws Exception {
```
Comment (gpang) [inline servers/src/test/java/tachyon/master/next/filesystem/meta/InodeFileTests.java]: is the fullyInMemoryTest implemented yet in InodeFile?

File: servers/src/test/java/tachyon/master/next/filesystem/meta/AbstractInodeTests.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+/**
+ * Abstract class for serving inode tests.
+ */
+public abstract class AbstractInodeTests {
+
+  @Rule
+  public ExpectedException mThrown = ExpectedException.none();
+
+  protected long createBlockId(long containerId) {
```
Comment (gpang) [inline servers/src/test/java/tachyon/master/next/filesystem/meta/AbstractInodeTests.java]: I think technically, this is creating an inode id for an InodeFile, right? If so, this should be createInodeFileId().

**Comment by gpang** [comment]
I just left a few minor comments. Thanks!

File: servers/src/test/java/tachyon/master/next/filesystem/meta/InodeFileTests.java (hunk 1 above)
Comment (yupeng9) [inline servers/src/test/java/tachyon/master/next/filesystem/meta/InodeFileTests.java]: Not fully implemented yet. This is a vanilla version that only tests the edge cases that we have implemented so far.

File: servers/src/test/java/tachyon/master/next/filesystem/meta/AbstractInodeTests.java (hunk 2 above)
Comment (yupeng9) [inline servers/src/test/java/tachyon/master/next/filesystem/meta/AbstractInodeTests.java]: Correct. I'm fine with renaming it for better.

**Comment by gpang** [comment]
LGTM
